src/ng-select/themes/bootstrap4.theme.scss

```diff
@@ -0,0 +1,381 @@
+@import "bootstrap/scss/bootstrap";
+@import "@ng-select/ng-select/scss/mixins";
+
+.form-control.ng-invalid {
+  @extend .form-control, .is-invalid;
+}
+
+.default-theme {
+  @import "@ng-select/ng-select/scss/default.theme.scss";
+}
+
+//$ng-select-highlight:  #007eff !default;
+$ng-select-primary-text: $input-color !default;
+$ng-select-disabled-text: $text-muted !default;
+$ng-select-border: $input-border-color !default;
+$ng-select-bg: $input-bg !default;
+$ng-select-selected: $dropdown-link-active-bg !default;
+$ng-select-marked: $dropdown-link-active-bg !default;
+
+.bootstrap-theme {
```

Comment:
This rule is something that doesn't exist in other theme files. You should change that in the same way as in other themes to be consistent.